src/styles/blocks/phone.scss

```diff
@@ -0,0 +1,30 @@
+@use "../utils/mixins" as *;
+
+.phone {
+  display: grid;
```

Comment:
The .phone class is set to display: grid; but no grid-template or grid-auto properties are defined. It's not clear how the grid is supposed to behave.